docs/user/exercises/programming-exercise-setup.rst

```diff
@@ -0,0 +1,110 @@
+Exercise Setup
+==============
+
+Exercise Creation
+-----------------
+
+1. **Open Course Management**
+
+- Open **Course Management**
+- Navigate into **Exercises** of your preferred course
+
+    .. figure:: programming/course-management-course-dashboard.png
+              :align: center
+
+2. **Generate programming exercise**
+
+- Click on **Generate new programming exercise**
+
+    .. figure:: programming/course-management-exercise-dashboard.png
+              :align: center
+
+- Fill out all required values and click on **Generate**
```

Comment:
Since this is a user guide, it would be good if you could provide some explanation into what the specific options are for, especially for things linke run tests once after due date, manual revi, mode, sequential test runs etc.

Reply:
We will include more details about certain features in the future. For the moment we want to replace the static PDF file with this doc :)

Reply:
I added two tables giving an overview over the supported features. More information can be provided in a follow up.